README-2.md

```diff
@@ -0,0 +1,125 @@
+# Concepts
```

Comment:
maybe just name this concepts.md? I kindof like how auto-value treat these breakouts as normal files https://raw.githubusercontent.com/google/auto/master/value/userguide/index.md